src/components/AddOrEditSkillModale.tsx

```diff
@@ -83,7 +88,11 @@ const AddOrEditSkillModale = ({
             {[1, 2, 3, 4, 5].map((index) => (
               <button
                 key={`desire-${index}`}
-                onClick={() => setDesireLevel(index)}
+                onClick={() => {
+                  setDesireLevel(index);
+                  if (skillLevel > 0 && index > 0)
+                    setautoValidation(true)
```

Comment:
Can't `onAddButtonClick` be directly called here?

Reply:
`setDesireLevel` is not synchronous, `onAddButtonClick` would use the old desire level

Reply:
Right!